site/content/docs/4.3/components/dropdowns.md

```diff
@@ -560,12 +560,13 @@ You can also create non-interactive dropdown items with `.dropdown-item-text`. F
 
 ### Active
 
-Add `.active` to items in the dropdown to **style them as active**.
+Add `.active` to items in the dropdown to **style them as active**
+Regarding accessibility, the active state should also be indicated with the `aria-current` attribute — using the `page` value for current page, or `true` for current item in a set.
```

Comment:
This doesn't add a newline. It might be better to add a `.` and continue on the same line